src/content-helper/dashboard-page/pages/traffic-boost/store.ts

```
@@ -0,0 +1,735 @@
+					...state,
+					inboundLinksTab: {
+						...state.inboundLinksTab,
+						links: remainingLinks,
+					},
+					selectedLink: action.updateSelectedLink && state.selectedLink?.uid === action.link.uid
+						? remainingLinks[ 0 ] ?? null
+						: state.selectedLink,
+				};
+			case 'UPDATE_SUGGESTION':
+				const isMatchingSuggestion = ( suggestion: TrafficBoostLink ) =>
+					suggestion.uid === action.suggestion.uid;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)